state-chain/pallets/cf-broadcast/src/lib.rs

```diff
@@ -467,82 +423,14 @@ pub mod pallet {
 
 	#[pallet::call]
 	impl<T: Config<I>, I: 'static> Pallet<T, I> {
-		/// Submitted by the nominated node when they cannot sign the transaction.
-		/// This triggers a retry of the signing of the transaction
-		///
-		/// ## Events
-		///
-		/// - N/A
-		///
-		/// ## Errors
-		///
-		/// - [InvalidBroadcastAttemptId](Error::InvalidBroadcastAttemptId)
-		/// - [InvalidSigner](Error::InvalidSigner)
+		/// DEPRECATED. This call is no longer used.
```

Comment:
Dan added this - I would prefer to replace the extrinsic too - I think this is more like a coding-style / maintenance rule type of discussion that we should make a decision on (this will undoubtedly happen more in the future)

Reply:
(and by "we" I mean the dev managers, like Dan, Martin and you. I do not have a strong feeling against either way, except that I just think it's "ugly" to leave this deprecated call here. 
Also I think the "call_index" macro is specifically there to cover for this situation)

Reply:
Ok, let's leave it for now, if Dan's added that, and we can look at removing it later